packages/codebytes/src/index.tsx

```diff
@@ -37,12 +37,17 @@ const EditorContainer = styled(Background)<EditorStyleProps>(
 export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
   text: initialText,
   language: initialLanguage,
-  hideCopyButton,
+  hideCopyButton = false,
   isIFrame = false,
   snippetsBaseUrl = process.env.CONTAINER_API_BASE,
+  onEdit,
+  onLanguageChange,
+  onCopy,
 }) => {
-  const [text, setText] = useState<string>(initialText);
-  const [language, setLanguage] = useState<LanguageOption>(initialLanguage);
+  const [text, setText] = useState<string>(initialText ?? '');
+  const [language, setLanguage] = useState<LanguageOption>(
+    initialLanguage ?? ''
+  );
```

Comment:
The static sites tests affirmed this desired behavior: if we don't have an initialLanguage or initialText, we should pass an empty string